test/unit/resources/aws_iam_password_policy_test.rb

```diff
@@ -66,6 +66,13 @@ def test_number_of_passwords_to_remember_returns_configured_value
     assert_equal expectedValue, AwsIamPasswordPolicy.new(@mockConn).number_of_passwords_to_remember
   end
 
+  def test_policy_to_s
+    configure_policy_password_reuse_prevention(value: Object.new)
+    expectedValue = "IAM Password-Policy"
+    test = AwsIamPasswordPolicy.new(@mockConn).to_s[0..expectedValue.length - 1]
```

Comment:
Why do you need to take a subset of the returned string?  I ask because this looks like it could create false-success: if **to_s** returned 'IAM Password-Policy FAIL FAIL FAIL', then the test as-written would pass, no?